CONTRIBUTING.rst

```diff
@@ -0,0 +1,79 @@
+--------------------
+Contributing to PUDL
+--------------------
+
+Welcome! We're so glad you're interested in contributing to PUDL! We would love
+some help making PUDL data as complete as possible.
+
+.. _after-intro:
+
+.. IMPORTANT:: Already have a dataset in mind?
+
+   If you **need data that's not in PUDL** that we're missing in PUDL,
+   `open an issue <https://github.com/catalyst-cooperative/pudl/issues/new/choose>`__.
```

Comment:
Is there a way to point to the 'new data' issue template here?

Reply:
Yes, but only after the issue template is made I think!

Reply:
Wait, I can just add a speculative link.